app/models/user_relation.rb

```diff
@@ -0,0 +1,14 @@
+class UserRelation 
+    attr_reader :url, :username
+    def initialize(aspects)
+        @url = aspects["html_url"]
+        @username = aspects["login"]
+    end 
+
+    def self.generate(raw_data)
+        parsed_data = JSON.parse(raw_data.body)
+        parsed_data.map do |relation|
+            test = UserRelation.new(relation) 
```

Comment:
we could probably get rid of the test =

Reply:
All these tiny oversights are embarrassing